src/Mod/TechDraw/Gui/QGIPrimPath.h

```diff
@@ -88,32 +85,25 @@ class TechDrawGuiExport QGIPrimPath : public QGraphicsPathItem
     virtual QColor getNormalColor();
     virtual QColor getPreColor();
     virtual QColor getSelectColor();
+    virtual QColor getDefaultFillColor() {
+        return Qt::white;
```

Comment:
Parameters FaceColor or PageColor in Mod/TechDraw/Colors?  Doesn't seem to be a preference getter for FaceColor.

Reply:
Not sure; this just extracts the hard-coding in multiple places into one location